### ISSUE TITLE
Get error if   target_core_mod is built-in in the kernel

### ISSUE THREAD

**Issue by Edoa** [Datera/targetcli#10]
Hi, I got this error running targetcli if that module is builtin in the kernel and not compiled as a module:
```Kernel module target_core_mod does not exists on disk and is not loaded.```

Plus I get
``` * Target Core modules not found!```
Starting target init scrpit.

**Comment by tramjoe** [comment]
This usecase is not supported just yet. I started a thread on the mailing-list to check if we should support builtin modules and if this does not cause any problems. If the answer is yes, I'll update the userspace accordingly.

**Comment by pjschmitt** [comment]
I looked at the mailing list thread and saw the comment about "no distros ship with it built in". Not true, most Gentoo users build their own kernels, that's somewhat the standard.

**Comment by ScottESanDiego** [comment]
Agreed.  Requiring this as a module doesn't seem reasonable for many use cases.
